src/relatedContent.ts

```diff
@@ -47,7 +55,7 @@ const parseHeaderImage = (content: Content): Image | undefined => {
 					url: asset.file ?? '',
 					height: asset.typeData?.height ?? 360,
 					width: asset.typeData?.width ?? 600,
-					alt: data?.alt,
+					altText: data?.alt,
```

Comment:
I changed the property name because the type `Image` has `altText` as a property. And we were setting the `alt` instead which doesn't exist in `Image`

Reply:
Good catch .. I'm surprised typescript wasn't complaining before .. I wonder why 🤔